### PR TITLE
ENT-5306 Add platform names to packages

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by Lex-2008** [comment]
@cf-bottom exotics build it please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4447/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4447/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4447/

**Comment by Lex-2008** [comment]
rebuild with all platforms: https://ci.cfengine.com/job/pr-pipeline/4477/
packages will appear here: http://buildcache.cloud.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4477/

File: build-scripts/package [hunk 1]
```diff
@@ -240,8 +240,13 @@ case "$PACKAGING" in
       DEB_VERSION="$VERSION~$BUILD_NUMBER"
     fi
 
+    # on debian/ubuntu, $OS_VERSION contains too many detials.
+    # leave only major version:
+    # 7.0 => 7
+    # 16.04 => 15
+    short_os_version="${OS_VERSION%%.*}"
```
Comment (nickanderson) [inline build-scripts/package]: nit pick: I would call this os_version_major not that it matters a whole lot.

File: packaging/cfengine-community/cfengine-community.spec.in [hunk 2]
```diff
@@ -3,7 +3,7 @@
 Summary: The CFEngine Configuration System
 Name: cfengine-community
 Version: @@VERSION@@
-Release: @@RELEASE@@
+Release: @@RELEASE@@%{?dist}
```
Comment (nickanderson) [inline packaging/cfengine-community/cfengine-community.spec.in]: what does this do?

Comment (vpodzime) [reply packaging/cfengine-community/cfengine-community.spec.in]:
```
[vpodzime@vpodzime-laptop ~]$ rpm --eval '%{dist}'
.fc30
```

File: build-scripts/package [hunk 3]
```diff
@@ -240,8 +240,13 @@ case "$PACKAGING" in
       DEB_VERSION="$VERSION~$BUILD_NUMBER"
     fi
 
+    # on debian/ubuntu, $OS_VERSION contains too many detials.
```
Comment (vpodzime) [inline build-scripts/package]: *detials*

File: build-scripts/package [hunk 4]
```diff
@@ -240,8 +240,13 @@ case "$PACKAGING" in
...
+    # leave only major version:
+    # 7.0 => 7
+    # 16.04 => 15
```
Comment (vpodzime) [inline build-scripts/package]: *16.04 => 15* :wink:

**Pull request merged**